cli/src/app.rs

```diff
@@ -232,16 +232,17 @@ pub fn add_subcommands(command: Command) -> Command {
         .subcommand(Command::new("ping").about("Ping the remote system to verify it is available"))
         .subcommand(
             Command::new("parse").about("Parse a lockfile and output its packages as JSON").args(&[
-                Arg::new("LOCKFILE")
+                Arg::new("lockfile")
                     .value_name("LOCKFILE")
                     .value_hint(ValueHint::FilePath)
-                    .help("The package lock file to submit."),
+                    .help("The package lock file to submit.")
+                    .action(ArgAction::Append),
                 Arg::new("lockfile-type")
                     .short('t')
                     .long("lockfile-type")
                     .value_name("type")
                     .requires("LOCKFILE")
```

Comment:
```suggestion
                    .requires("lockfile")
```
This change will fix the broken test.